Model/ContactClientModel.php

```diff
@@ -306,22 +306,28 @@ public function getStats(
         $dateFormat = null,
         $canViewOthers = true
     ) {
-        $unit           = (null === $unit) ? $this->getTimeUnitFromDateRange($dateFrom, $dateTo) : $unit;
-        $dateToAdjusted = clone $dateTo;
-        if (in_array($unit, ['H', 'i', 's'])) {
-            // draw the chart with the correct intervals for intra-day
-            $dateToAdjusted->setTime(23, 59, 59);
+        if ($dateFrom->getTimezone()->getName() !== 'UTC') {
+            $dateFrom->setTimezone(new \DateTimeZone('UTC'));
```

Comment:
Resetting to UTC invalidates the user's timezone. You should not need to do this.

Reply:
it was being done below this. I just updated the logic to do so?